style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
- [ ] Kindly declare your styling using the `class` given to the `h1` as stated in the project requirements 🙏

Reply:
Thanks! It's done